main/variables.tf

```diff
@@ -0,0 +1,206 @@
+  default     = 10
+  type        = "string"
+}
+
+variable "storage-type" {
+  description = "RDS storage type."
+  default     = "gp2"
+  type        = "string"
+}
+
+variable "instance-class" {
+  description = "RDS instance class."
+  default     = "db.t2.micro"
+  type        = "string"
+}
+
+variable "engine-version" {
+  description = "RDS engine version."
+  default     = "8.0"
+  type        = "string"
+}
+
+variable "parameter-group-name" {
+  description = "RDS parameter group name."
+  default     = "default.mysql8.0"
+  type        = "string"
+}
+
+variable "username" {
+  description = "RDS username."
+  default     = "bdp"
+  type        = "string"
+}
+
+variable "password" {
+  description = "RDS password."
+  default     = "Ch4ll3ng3"
+  type        = "string"
+}
+
+# SQS specific
+
+variable "queue-name" {
+  description = "Queue name."
+  default     = "beekeeper-apiary-listener"
+  type        = "string"
+}
+
+variable "message-retention-seconds" {
+  description = "SQS message retention (s)."
+  default     = "604800"
+  type        = "string"
+}
+
+variable "receive-wait-time-seconds" {
+  description = "SQS receive wait time (s)."
+  default     = "20"
+  type        = "string"
+}
+
+variable "apiary-metastore-listener-arn" {
+  description = "ARN of the Apiary Metastore Listener."
+  type        = "string"
+}
+
+# ECS specific
+
+variable "path-scheduler-apiary-docker-image-url" {
+  description = "URL to the Beekeeper path-scheduler image."
+  type        = "string"
+}
+
+variable "cleanup-docker-image-url" {
+  description = "URL to the Beekeeper cleanup image."
+  type        = "string"
+}
+
+variable "path-scheduler-apiary-memory" {
+  description = <<EOF
+The amount of memory (in MiB) used to allocate for the Beekeeper Path Scheduler Apiary container.
+Valid values: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html
+EOF
+
+  default = "4096"
+  type    = "string"
+}
+
+variable "path-scheduler-apiary-cpu" {
+  description = <<EOF
+The amount of CPU used to allocate for the Beekeeper Path Scheduler Apiary ECS task.
+Valid values: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html
+EOF
+
+  default = "2048"
+  type    = "string"
+}
+
+variable "cleanup-memory" {
+  description = <<EOF
+The amount of memory (in MiB) used to allocate for the Beekeeper Cleanup container.
+Valid values: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html
+EOF
+
+  default = "4096"
+  type    = "string"
+}
+
+variable "cleanup-cpu" {
+  description = <<EOF
+The amount of CPU used to allocate for the Beekeeper Cleanup ECS task.
+Valid values: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html
+EOF
+
+  default = "2048"
+  type    = "string"
+}
+
+variable "graphite-enabled" {
+  description = "Enable to produce Graphite metrics - true or false."
+  default     = "false"
```

Comment:
is it not possible to have this as a boolean ?

Reply:
I dont think thats a valid type in terraform 11

Reply:
oh ok.